powerlevel9k.zsh-theme

```diff
@@ -19,9 +19,17 @@
 
 # Check if the theme was called as a function.
 if [[ $(whence -w prompt_powerlevel9k_setup) =~ "function" ]]; then
-  # Script is a function! We assume this to happen only in
-  # prezto, as they use the zstyle-builtin to set the theme.
-  0="${ZDOTDIR:-$HOME}/.zprezto/modules/prompt/functions/prompt_powerlevel9k_setup"
+  autoload -U is-at-least
+  if is-at-least 5.0.8; then
+    # Try to find the correct path of the script.
+    0=$(whence -v $0 | sed "s/$0 is a shell function from //")
+  else
```

Comment:
You found exactly the last loophole. ;)
I haven't found a solution for someone who is using ZSH < 5.0.8 and not prezto. And I am afraid there is no solution for this case. Except for the workaround of setting the path by a variable (maybe if the file `${ZDOTDIR:-$HOME}/.zprezto/modules/prompt/functions/prompt_powerlevel9k_setup` is not present). But I think this is not a good solution..
But I should update the comments at least. They are indeed appropriate.